ostd/src/logger.rs

```diff
@@ -2,36 +2,60 @@
 
 //! Logging support.
 
+use alloc::format;
+
 use log::{Level, Metadata, Record};
+use owo_colors::OwoColorize;
```

Comment:
I would like to feature-gate the use of `owo_colors` to minimize the TCB of ostd. Let's add a feature named `log_color`, which depends on this `owo_colors` crate. This `log_color` feature can be enabled by default. When the feature is not enabled, use the current implementation.